build.gradle

```diff
@@ -1,80 +1,86 @@
 buildscript {
-    ext {
-        grailsVersion = project.grailsVersion
-    }
-    repositories {
-        mavenLocal()
-        maven { url "https://repo.grails.org/grails/core" }
-    }
-    dependencies {
-        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:2.8.2"
-    }
+	ext {
+		grailsVersion = project.grailsVersion
+	}
+	repositories {
+		mavenLocal()
+		maven { url "https://repo.grails.org/grails/core" }
+	}
+	dependencies {
+		classpath "org.grails:grails-gradle-plugin:$grailsVersion"
+		classpath "com.bertramlabs.plugins:asset-pipeline-gradle:2.15.1"
+	}
 }
 
 def setIfNotSet = { String name, value ->
-    if (!project.ext.has(name)) {
-        project.ext[name] = value
-    }
+	if (!project.ext.has(name)) {
+		project.ext[name] = value
+	}
 }
 setIfNotSet 'issueTrackerUrl', project.vcsUrl + '/issues'
 setIfNotSet 'websiteUrl', project.vcsUrl
 
-version "1.2.1.BUILD-SNAPSHOT"
+version "1.3.0-BUILD-SNAPSHOT"
 group = 'org.grails.plugins'
 
-apply plugin:"eclipse"
+apply plugin: "eclipse"
 apply plugin: "idea"
 apply plugin: "org.grails.grails-plugin"
 apply plugin: "org.grails.grails-plugin-publish"
 apply plugin: "org.grails.grails-gsp"
 apply plugin: "asset-pipeline"
 
 ext {
-    grailsVersion = project.grailsVersion
-    gradleWrapperVersion = project.gradleWrapperVersion
+	grailsVersion = project.grailsVersion
+	gradleWrapperVersion = project.gradleWrapperVersion
 }
 
 repositories {
-    mavenLocal()
-    maven { url "https://repo.grails.org/grails/core" }
+	mavenLocal()
+	maven { url "https://repo.grails.org/grails/core" }
 }
 
 dependencyManagement {
-    imports {
-        mavenBom "org.grails:grails-bom:$grailsVersion"
-    }
-    applyMavenExclusions false
+	imports {
+		mavenBom "org.grails:grails-bom:$grailsVersion"
+	}
+	applyMavenExclusions false
 }
 
 dependencies {
-    provided 'javax.servlet:javax.servlet-api:3.1.0'
-    provided 'org.grails:grails-dependencies'
-    compile 'org.codehaus.groovy:groovy-all:2.4.3'
-    provided 'org.grails.plugins:spring-security-core:3.+'
-    provided 'org.grails.plugins:spring-security-oauth2:1.2.0+'
+	provided 'javax.servlet:javax.servlet-api:3.1.0'
+	provided 'org.grails:grails-dependencies'
 
-    console "org.grails:grails-console"
+	provided 'org.grails.plugins:spring-security-core:3.+'
+	provided 'org.grails.plugins:spring-security-oauth2:1.3.0+'
```

Comment:
Need to release the `grails-plugins/grails-spring-security-oauth2:1.3.0` plugin first before this one works